README.windows.md

```diff
@@ -121,7 +121,9 @@ or edit `%USERPROFILE%\.gitconfig` and add/edit the lines:
     make -j 4   # Adjust the number of cores (4) to match your build environment.
 ```
 
-5. Setup Package Development Environment
+  5. There's several windows specific tools, like a portable installation of git, which are included when you install Julia using the installer, but are not built with make above.  If you need these you can run ```make win-extras``` and move things around by hand, or run ```make binary-dist``` and run the installer that it generates.  
```

Comment:
"There are several"

Also delete the trailing whitespace, otherwise this will fail `make check-whitespace`.